### PR TITLE
feature: Add new API /create param `loginURL`

### PR DESCRIPTION
**Pull request by gustavotrott**
This pull request adds a new parameter `loginURL` to the `/create` API endpoint. The purpose of this parameter is to allow third-party applications to specify a URL for users to access and join a live meeting.

This feature is particularly useful in hybrid settings. For example, a teacher can display a QR code within BigBlueButton (BBB), which students can scan to be taken directly to a site like Greenlight. There, they can enter their name and join the meeting to participate in activities like an online poll.

API /create param:
`loginURL=https://myserver.bbb//rooms/4h9-3s2-ffs-mam`

Graphql query:
```gql
subscription {
  meeting {
    loginUrl
  }
}
```

### CODE REVIEW

**Comment by gustavotrott** [comment]
Docs is missing! It will be provided in a subsequent PR.

File: bbb-common-web/src/main/java/org/bigbluebutton/api/domain/Meeting.java [hunk 1]
```diff
@@ -143,6 +144,7 @@ public Meeting(Meeting.Builder builder) {
         maxUsers = builder.maxUsers;
         bannerColor = builder.bannerColor;
         bannerText = builder.bannerText;
+		loginUrl = builder.loginUrl;
```
Comment (antobinary) [inline bbb-common-web/src/main/java/org/bigbluebutton/api/domain/Meeting.java:147]:
```suggestion
	loginUrl = builder.loginUrl;
```

File: akka-bbb-apps/src/main/scala/org/bigbluebutton/core/db/MeetingDAO.scala [hunk 2]
```diff
@@ -7,6 +7,14 @@ import org.bigbluebutton.core.apps.groupchats.GroupChatApp
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.util.{ Failure, Success }
 
+case class MeetingSystemColumnsDbModel(
+      loginUrl:                              Option[String],
+      logoutUrl:                             String,
```
Comment (antobinary) [inline akka-bbb-apps/src/main/scala/org/bigbluebutton/core/db/MeetingDAO.scala:12]: Why is `logoutUrl` not `Option[String]`?

**Review by antobinary — APPROVED** [review]
LGTM. Let's wait for the CI too though.

**Comment by antobinary** [comment]
Here's Greenlight passing the parameter (request) https://github.com/bigbluebutton/greenlight/issues/5802

**Pull request merged**